pallets/stableswap/src/lib.rs

```diff
@@ -179,15 +179,14 @@ pub mod pallet {
 			pool_id: T::AssetId,
 			who: T::AccountId,
 			shares: Balance,
-			assets: Vec<AssetLiquidity<T::AssetId>>,
+			assets: Vec<AssetBalance<T::AssetId>>,
 		},
 		/// Liquidity removed.
 		LiquidityRemoved {
 			pool_id: T::AssetId,
 			who: T::AccountId,
 			shares: Balance,
-			asset: T::AssetId,
-			amount: Balance,
+			amounts: Vec<AssetBalance<T::AssetId>>,
```

Comment:
This change is not necessary

Reply:
it is preparation for remove_liquidity where you obtain all assets. This functionality will be added later. 

so just to make sure that if anything/anyone already works with events, there wont be breaking changes.